.stylelintrc.js

```diff
@@ -22,5 +22,13 @@ module.exports = {
     ],
     // Would be valuable for strict BEM components but is too hard to enforce with legacy code.
     'no-descending-specificity': null,
+    // Override stylelint-config-wagtail’s options to allow all float and clear values for now.
+    'declaration-property-value-allowed-list': {
+      // 'clear': ['both', 'none'],
+      // 'float': ['inline-start', 'inline-end', 'none', 'unset'],
+      'text-align': ['start', 'end', 'center'],
+    },
+    // Disable declaration-strict-value until we are in a position to enforce it.
```

Comment:
Is the plan to use a newer version of the Wagtail stylelint LIbrary? - let me know if you need me to do a new release of that library to help. 

Ignore this if you were just adding this temporarily.

Reply:
Ok. Thanks

Reply:
Ok. Thanks